test/e2e/pods/pods.go

```diff
@@ -0,0 +1,49 @@
+package pods
+
+import (
+	"bytes"
+	"os"
+
+	testclient "github.com/openshift/ingress-node-firewall/test/e2e/client"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+// ExecCommand runs command in the pod and returns buffer output
+// copied entirely from SRIOV Network Operator
+func ExecCommand(cs *testclient.ClientSet, pod *corev1.Pod, command ...string) (string, string, error) {
```

Comment:
Nice that will save me from the trouble of trying to use kubectl exec Thanks for adding this !!

Reply:
Thank the SRIOV network operator folks. I see nearly the exact same function in upstream k8 code so maybe I should thank them too ;)